Add demo links in footer

owldock opened the pull request:
Per #33  Added links in footer to:
-Payment Request Demo
-Crowdfund App Demo
-Store Demo

first pull request, so be gentle :)

owldock committed:
Add demo links in footer

Added links in footer to:
-Payment Request Demo
-Crowdfund App Demo
-Store Demo

pavlenex commented on index.html:
```diff
@@ -39,7 +39,7 @@
 		c-0.113-3.857,23.594-17.424,25.74-15.641s2.854,29.277,0,31.28S921.227,46.965,921.113,43.107z"/><path id="smtri4_1_" opacity="0.15" fill="#1A001A" enable-background="new    " d="M884.113,98.107
 		c-0.113-3.857,23.594-17.424,25.74-15.641s2.854,29.277,0,31.28S884.227,101.965,884.113,98.107z"/></g></svg><div class="blockGreenFoot"><div class="footerBlock"><div><h3 class="ftbTr_" id="fBTrig_-0">BTCPAY BASICS</h3><ul class="ftb_" id="fBT_-0"><li><a alt="Introduction" href="https://docs.btcpayserver.org/">Introduction</a></li><li><a alt="Use Case" href="https://docs.btcpayserver.org/btcpay-ba ...
-                                    href="https://docs.btcpayserver.org/btcpay-basics/btcpayvsothers">BTCPay vs others</a></li><li><a alt="Lightning Network" href="https://docs.btcpayserver.org/features/lightningnetwork">Lightning Network</a></li><li><a alt="Try it Out" href="https://mainnet.demo.btcpayserver.org/">Try it Out</a></li></ul></div><div><h3 class="ftbTr_" id="fBTrig_-1">DEPLOYMENT</h ...
+                                    href="https://docs.btcpayserver.org/btcpay-basics/btcpayvsothers">BTCPay vs others</a></li><li><a alt="Lightning Network" href="https://docs.btcpayserver.org/features/lightningnetwork">Lightning Network</a></li><li><a alt="Try it Out" href="https://mainnet.demo.btcpayserver.org/">Try it Out</a></li></ul></div><div><h3 class="ftbTr_" id="fBTrig_-1">DEPLOYMENT</h ...
```
The actual link to the store demo has now been changed to https://store.btcpayserver.org, can you change it? The rest looks good.

vswee replied:
Does that mean that all occurrences of `https://mainnet.demo.btcpayserver.org` including `https://mainnet.demo.btcpayserver.org/``sub-directory/` need updating?

pavlenex replied:
Nope, just store.demo.btcpayserver.org does not work, instead, site has been migrated store.btcpayserver.org

owldock replied:
I couldn't get the Payment Button Demo link to resolve -- neither https://store.demo.btcpayserver.org/index.php/payment-button-demo and https://store.btcpayserver.org/payment-button-demo worked as links. And has the Crowdfund App linked changed too? That one doesn't resolve now either (https://btcpay.kukks.org/apps/pMb1bieuysP3MH2d2s9Yi11Pvb/crowdfund)

pavlenex replied:
@owldock Yes, I think we have to ditch payment button demo until we have  a page for it, will open a separate issue. So just disregard payment button demo, the crowdfunding link works for me though?

owldock replied:
OK cool -- crowdfund link works for me now too, updated the index.html to remove "payment button demo"

owldock committed:
Update index.html

pavlenex commented:
Can you resolve the conflicts?

owldock committed:
Merge branch 'master' into master

owldock commented:
> Can you resolve the conflicts?

I think I did that right, but first time resolving, happy to amend if not. Thanks for your patience! 🙌

vswee commented:
@owldock I'm hoping you'll be able to replicate this in [r/source/src/html/footer-tmpl.html](https://github.com/btcpayserver/btcpayserver.github.io/blob/master/source/src/html/footer-tmpl.html) so that it can built across all language versions :)

owldock committed:
Added Payment Request + Crowdfund App to footer template

first time w/Transifex, but I added the new phrases "Crowdfund App Demo" and "Payment Request Demo" to the end of en.json and referred to them in the src/html/tmpl.html with the appropriate tags

owldock committed:
Merge branch 'master' into master
